docs/advanced-hyperopt.md

```diff
@@ -0,0 +1,63 @@
+# Advanced Hyperopt
+
+This page explains some advanced Hyperopt issues that may require higher
```

Comment:
> This page explains some advanced Hyperopt issues that ...

i would not call it "issues" - wanting to customize a loss function should not start (nor end) in an issue ...
more like "advanced Hyperopt usecases" - or "advanced Hyperopt applications" (?) ... 
Not necessarily one of the 2 above if you have a better idea...

Reply:
pls review my suggestion for this